pkg/chartutil/create.go

```diff
@@ -92,6 +106,36 @@ const defaultIgnore = `# Patterns to ignore when building packages.
 *.tmproj
 `
 
+const defaultIngress = `{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+  app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+    {{- range $key, $value := .Values.ingress.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+    - host: {{ .Values.ingress.hostname }}
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: {{ template "fullname" . }}
+              servicePort: 80
```

Comment:
@archSeer I like the `$serviceName` and `$servicePort` definitions 👍 I Just pushed a change (2be4547) that uses the changes in your template above, except I kept `path: /`.

Was leaving that our an oversight, or is there a reason we do not want this? Or should we parameterize this as part of the hosts array, like this?
```yaml
ingress:
  hosts:
    - hostname: chart-example.local
      path: "/"
```